test/cri-containerd/main.go

```diff
@@ -87,6 +87,15 @@ var allFeatures = []string{
 	featureGPU,
 }
 
+type config struct {
```

Comment:
might as well get rid of this: https://github.com/dcantah/hcsshim/blob/test-config-decl/test/cri-containerd/removepodsandbox_test.go#L14

Reply:
I didn't even realize you'd already done this *facepalm*. Could change your tests use this or vice versa, doesn't matter too much. Let's see if we even want to do this anyways first

Reply:
Either way, we should make sure the struct is defined in main.go